admin/data/sql/migrations/20200205101232.do.pupil-attendance-indexes.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX idx_pupilAttendance_attendanceCode_id ON mtc_admin.[pupilAttendance] (attendanceCode_id) WITH (DROP_EXISTING = ON)
+CREATE INDEX idx_pupilAttendance_pupil_id ON mtc_admin.[pupilAttendance] (pupil_id) WITH (DROP_EXISTING = ON)
```

Comment:
This index will not apply to my system, as the expected index does not already exist which it expects to find and drop.  The first index in this file already exists, so I am not sure why we are adding another.

It would seem safer to use the FIND / DROP / CREATE pattern that the previous index creations are using.

Reply:
the `DROP_EXISTING` construct is gone, as it throws an error if it doesn't already exist.
see updated - all using common construct